_inc/client/state/survey/actions.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import restApi from 'rest-api';
+import {
+	JETPACK_MARKETING_SUBMIT_SURVEY,
+	JETPACK_MARKETING_SUBMIT_SURVEY_SUCCESS,
+	JETPACK_MARKETING_SUBMIT_SURVEY_FAIL,
+} from 'state/action-types';
+
+const dispatchSurvey = surveyResponse => dispatch => {
+	dispatch( {
+		type: JETPACK_MARKETING_SUBMIT_SURVEY,
+	} );
+
+	return restApi
+		.submitSurvey( surveyResponse )
+		.then( data => {
+			dispatch( {
+				type: JETPACK_MARKETING_SUBMIT_SURVEY_SUCCESS,
+			} );
+
+			return data;
+		} )
+		.catch( error => {
+			dispatch( {
+				type: JETPACK_MARKETING_SUBMIT_SURVEY_FAIL,
+				error: error.response,
+			} );
+		} );
+};
+
+export const submitSurvey = ( siteId, sitePlan, surveyAnswerId, surveyAnswerText, location ) => {
+	const surveyResponse = {
+		survey_id: 'calypso-disconnect-jetpack-july2019',
```

Comment:
Should `july2019` be updated here to reflect our slightly revised launch date?

Reply:
I will look into the difference, but I think this would just group the answers with the same survey from calypso

Reply:
also
> july2019
😅

Reply:
going to keep it since same id is still used in calypso